src/pages/projectSettings/tabs/GithubCommitQueueTab/getFormSchema.tsx

```diff
@@ -212,60 +213,83 @@ export const getFormSchema = (
                     enabled: {
                       enum: [true],
                     },
-                    mergeQueueTitle: {
-                      title: "Merge Queue",
-                      type: "null",
-                    },
-                    mergeQueue: {
-                      type: "string" as "string",
-                      oneOf: [
-                        {
-                          type: "string" as "string",
-                          title: "Evergreen",
-                          enum: [MergeQueue.Evergreen],
-                          description:
-                            "Use the standard commit queue owned and maintained by Evergreen.",
-                        },
-                        {
-                          type: "string" as "string",
-                          title: "GitHub",
-                          enum: [MergeQueue.Github],
-                          description: "Use the GitHub merge queue.",
-                        },
-                      ],
-                    },
                     message: {
                       type: "string" as "string",
                       title: "Commit Queue Message",
                     },
-                    mergeMethod: {
-                      type: "string" as "string",
-                      title: "Merge Method",
-                      oneOf: [
-                        {
-                          type: "string" as "string",
-                          title: "Squash",
-                          enum: ["squash"],
-                        },
-                        {
+                    mergeSettings: {
+                      title: "Merge Queue",
+                      type: "object" as "object",
+                      properties: {
+                        mergeQueue: {
                           type: "string" as "string",
-                          title: "Merge",
-                          enum: ["merge"],
+                          oneOf: [
+                            {
+                              type: "string" as "string",
+                              title: "Evergreen",
+                              enum: [MergeQueue.Evergreen],
+                              description:
+                                "The standard commit queue owned and maintained by Evergreen.",
```

Comment:
```suggestion
                                "Evergreen's commit queue.",
```
I'd suggest at least striking "standard", since the goal is to replace it with GitHub's, making that one the standard one. I suggested a simpler phrase to mirror the GitHub one.

Reply:
done! I wonder if I should say anything about the CLI only working with the evg one? :0

Reply:
I think that because there's a button in the GitHub UI, and also it's explained in the linked docs, that this is enough.